### DISCUSSION TITLE
A youtube script generator

### DISCUSSION THREAD

**Discussion by Dec-35** [openai/whisper#1017] [Show and tell]
### I made this around whisper for Python
This uses selenium for web scraping, DeepL for translation and Eel for the html UI.

It can either use a video link
![image](https://user-images.githubusercontent.com/88966922/222597169-15ae3d24-f778-4d50-a45a-8b79a716d21c.png)
![image](https://user-images.githubusercontent.com/88966922/222597263-f0a442c9-fa17-402e-b51a-7d6ad54b5ca0.png)
![image](https://user-images.githubusercontent.com/88966922/222597876-1f59669f-91a3-435d-9f66-9555ebb18382.png)

Or it can use a youtube channel and process every not-already-processed videos
![image](https://user-images.githubusercontent.com/88966922/222597989-ef514d0f-ac9c-4e49-9049-f3133d47a6a4.png)

**Comment by mjtechguy** [comment]
Interested in this. Do you have a repo?

**Reply by Dec-35** [reply]
I did but sadly there was a miscommunication with my employer and I had to abandon the project. (I wasn't getting paid anymore)
